wrappers/python/tests/model/test_bert_embeddings.py

```diff
@@ -0,0 +1,210 @@
+            torch_model, params.batch_size, params.batch_size_tile,
+            params.seq_len, params.seq_len_tile, nntile_config, 0)
+    nntile_model.clear_gradients()
+    x_random = gen.integers(params.seq_len,
+                            size=nntile_model.activations[0].value.shape,
+                            dtype=np.int64)
+
+    x_nntile = np.array(x_random, dtype=np.int64, order='F')
+    nntile_model.activations[0].value.from_array(x_nntile)
+    x_torch = torch.tensor(x_nntile.T)
+    y_grad_random = gen.standard_normal((params.hidden_size,
+                                         params.seq_len,
+                                         params.batch_size),
+                                         dtype=np.float32)
+    y_grad_nntile = np.array(y_grad_random, dtype=np.float32, order='F')
+    nntile_model.activations[-1].grad.from_array(y_grad_nntile)
+    y_grad_torch = torch.Tensor(y_grad_nntile.T)
+    return torch_model, nntile_model, x_torch, y_grad_torch
+
+
+@pytest.mark.parametrize('params', [
+    pytest.param(single_tile, id='single_tile'),
+    pytest.param(multiple_tiles, id='multiple_tiles'),
+])
+@pytest.mark.parametrize('dtype', [
+    'fp32',
+    pytest.param('bf16', marks=nocuda),
+    pytest.param('fp32_fast_tf32', marks=nocuda),
+    pytest.param('fp32_fast_fp16', marks=nocuda),
+    pytest.param('fp32_fast_bf16', marks=nocuda),
+])
+class TestBertEmbeddings:
+    def test_coercion(self, starpu_simple, torch_rng,
+                      params: BertTestParams,
+                      dtype: str):
+
+        torch_model, nntile_model, _, _ = generate_inputs(params, dtype)
+
+        torch_model_other = nntile_model.to_torch()
+        nntile_model.unregister()
+        rtol = dtype2tol[dtype]['rtol']
+        for (n1, p1), (n2, p2) in zip(torch_model.named_parameters(),
+                torch_model_other.named_parameters()):
+            assert n1 == n2
+            assert torch.norm(p1 - p2) <= rtol * torch.norm(p1)
+
+    def test_forward(self, starpu_simple, torch_rng,
+                     params: BertTestParams,
+                     dtype: str):
+        torch_model, nntile_model, x, _ = generate_inputs(params, dtype)
+        y_torch = torch_model(x)
+        nntile_model.forward_async()
+        y_nntile = torch.Tensor(to_numpy(nntile_model.activations[-1].value).T)
+        nntile_model.unregister()
+
+        rtol = dtype2tol[dtype]['rtol']
+        assert torch.norm(y_torch - y_nntile) <= rtol * torch.norm(y_torch)
+
+    def test_backward(self, starpu_simple, torch_rng,
+                      params: BertTestParams,
+                      dtype: str):
+        torch_model, nntile_model, x, y_grad = generate_inputs(params, dtype)
+        y = torch_model(x)
+        nntile_model.forward_async()
+        res = (y * y_grad).sum()
+        res.backward()
+
+        nntile_model.backward_async()
+        torch_model_other = nntile_model.to_torch_with_grads()
+        nntile_model.unregister()
+
```

Comment:
check gradient over input also

Reply:
Gradients for the int64 variables look strange

Reply:
indeed